R/helpers_knitr_Stopping.R

```diff
@@ -60,23 +60,31 @@ knit_print.StoppingMaxGainCIRatio <- function(
 #' @description `r lifecycle::badge("experimental")`
 #' @inheritParams knit_print.StoppingTargetProb
 #' @param preamble (`character`)\cr the text that introduces the list of rules
+#' @param indent (`integer`)\cr the indent level of the current stoppling rule
+#'   list. Spaces with length `indent * 4` will be padded to the beginning of
```

Comment:
"... will be prepended ..." is more natural than "... will be padded to...".  

Also, "stoppling" should be "stopping".  [Did you run `spelling::spell_check_package()`?]

Reply:
Thanks. I've updated the wording accrodingly. Oddly, `spelling::spell_check_package()` did not raise any info about "stoppling" though I did not know about this spell checking tool before.